packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -82,7 +83,7 @@ export default class extends EntityModel {
     const entityTypeModel = SYSTEM_TYPES.entityType.org;
 
     const entity = await EntityModel.create(graphApi, {
-      ownedById: systemAccountId,
+      ownedById: orgAccountId,
```

Comment:
This is a subtle but big change? 

I thought we'd agreed that we wanted all `User`s and `Org`s to live under the root account. I'm not against us changing this but I think it's worth us having a discussion on Slack

Reply:
Thanks for catching